geniza/corpus/templates/corpus/snippets/document_transcription.html

```diff
@@ -69,18 +71,32 @@ <h3>{{ image_info.shelfmark }} {{ image_info.label }}</h3>
                     </div>
                 </div>
                 <div class="transcription-panel">
-                    {% if forloop.first %}
-                        <h3>Transcription</h3>
-                        <span data-transcription-target="editionFullLabel" class="current-transcription">{{ document.digital_editions.0.display|safe }}</span>
+                    {% if image_info.not_in_document %}
+                        {# if this image isn't actually in this document, link to related documents #}
+                        <ul class="excluded-related">
```

Comment:
there will be some excluded images with no related documents; any problem with having an empty `<ul>`? should we structure this in a way that we know if we have any?

Reply:
Would it be OK to just `display: none` when it's empty?

Reply:
that sounds like a good solution to me!